### PR TITLE
added client usage in OGC API- Features

### PR DESCRIPTION
**Pull request by krishnaglodha**
Added examples of 
leaflet, openlayers, qgis, GDAL , Pygeoapi

### CODE REVIEW

File: workshop/content/docs/api-deep-dive/features.md [hunk 1]
```diff
@@ -174,6 +174,102 @@ the request
...
 
+### Client Usage
+
+Various clients/softwares supports OGC API- Features, You can checkout latest list [here](https://github.com/opengeospatial/ogcapi-features/blob/master/implementations/clients/README.md)
+
+In this workshop we'll check a client example ( leaflet and Openlayers ), Software example (QGIS), Native API example (GDAL) and also server example ( PygeoAPI )
+
+#### Leaflet Implementation
+
+Leaflet supports reading and implementing GeoJSON by default, thus just like any other  file or API which spits out data as GeoJSON, OGC API - Features can also be expose data as GeoJSON, by passing `f=json` in the string and then it can be utilized in Front end my adding following code block
```
Comment (tomkralidis) [inline workshop/content/docs/api-deep-dive/features.md:185]: - "via the front end by adding the following code:"
- always have colon on text before starting a code block

**Review by tomkralidis — CHANGES_REQUESTED** [review]
cc @doublebyte1

**Comment by krishnaglodha** [comment]
Maybe "Client" is wrong word , we can just write "Implementations", thoughts ?

File: workshop/content/docs/api-deep-dive/features.md [hunk 2]
```diff
@@ -174,17 +174,17 @@ the request
 A client application can then retrieve the GeoJSON document and display
 or process it.
 
-### Client Usage
+### Client usage
 
 Various clients/softwares supports OGC API- Features, You can checkout latest list [here](https://github.com/opengeospatial/ogcapi-features/blob/master/implementations/clients/README.md)
```
Comment (doublebyte1) [inline workshop/content/docs/api-deep-dive/features.md]: @krishnaglodha  We already have a list of implementations, [further up](https://github.com/opengeospatial/ogcapi-workshop/commit/1b45e2e36dcbdbaf77643d49addd953a2d4f93f5#diff-8fdc87e09dc02c51fa15700cf6d04ac3245d4c33377f84a405fd89d0bef54f67R63) on this page :) No need to repeat it, here!

Comment (tomkralidis) [reply workshop/content/docs/api-deep-dive/features.md]: Good catch, I think we wanted a specific "client" perspective. +1 to remove pygeoapi from the Client usage sections.

**Pull request merged**